pkg/lib/bundle/generate.go

```diff
@@ -33,6 +33,11 @@ type AnnotationMetadata struct {
 	Annotations map[string]string `yaml:"annotations"`
 }
 
+type ClusterServiceVersion struct {
```

Comment:
Unexport this since it is only used for unmarshalling?
```suggestion
type clusterServiceVersion struct {
```